app/main.py

```diff
@@ -16,4 +16,13 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+all_variables = [lucky_number, pi, one_is_a_prime_number, name,
+                 my_favourite_films, profile_info, marks, collection_of_coins]
+sorted_variables = {'mutable': [],
+                    'immutable': []}
+
+for variable in all_variables:
+    if isinstance(variable, set | dict | list):
+        sorted_variables['mutable'] += [variable]
+    else:
+        sorted_variables['immutable'] += [variable]
```

Comment:
you don't need a loop in this task. just add variables to the list in the value in the dictionary